### ISSUE TITLE
Round distances to a minimum of 10th

### ISSUE THREAD

**Issue by TobiPeterG** [organicmaps/organicmaps#9742]
⚠ Have you searched for similar, already existing issues? Yes

**Is your feature request related to a problem? Please describe.**
When I drive and stop at a traffic light, the distance to the intersection is "flickering" between e.g. 11m and 12m due to the inaccuracy of GPS

**Describe the ideal solution**
It would be nice if distances were rounded to a minimum of 10th, so e.g. instead of showing in 62m, just in 60m etc. This way, we're preventing distances from bouncing up and down

**Comment by map-per** [comment]
But with this approach we can still have flickering between e.g. 10m and 20m (if we are somewhere around 15m). Only updating the UI when the value differs Xm from the previous value sounds like a better solution to me.

**Comment by matkoniecz** [comment]
Maybe even better would be increasing distance only if it differs Xm from the previous value.

So when you cycle/walk/drive in a traffic jam the distance changes are not becoming choppy.

**Comment by TobiPeterG** [comment]
> But with this approach we can still have flickering between e.g. 10m and 20m (if we are somewhere around 15m). Only updating the UI when the value differs Xm from the previous value sounds like a better solution to me.

Yes, that is true. So maybe have it change on X7 and X3m, so when you're 24m away, it shows 30m, when you are 23m away, it shows 20m and only when you're 27m away again it would change to 30m. This leaves 24-26m as a wiggle room for GPS. :)
